maxim/logger/components/utils.py

```
@@ -0,0 +1,81 @@
+                image_url = item.get("image_url", "")
+                url = None
+                if image_url is not None and isinstance(image_url, dict):
+                    url = image_url.get("url", "")
+                elif image_url is not None and isinstance(image_url, str):
+                    url = image_url
+                if url is not None and url != "":
+                    # Check if its base64 encoded data uri
+                    if url.startswith("data:image"):
+                        # Extract base64 data from data URI
+                        match = re.match(
+                            r"data:image/(?P<ext>\w+);base64,(?P<data>.+)", url
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)